backend/stakepoold/rpc/rpcserver/context.go

```diff
@@ -354,6 +355,38 @@ func (ctx *AppContext) UpdateUserDataFromMySQL() error {
 	return nil
 }
 
+// returns a map of master pubkeys to their account
+func (ctx *AppContext) MyMasterPubKeys() (map[string]string, error) {
+	acctmap, err := ctx.WalletConnection.ListAccounts()
+	if err != nil {
+		log.Errorf("MyMasterPubKeys: ListAccounts rpc failed: %v", err)
+		return nil, err
+	}
+	keymap := make(map[string]string)
+	for acct := range acctmap {
+		// skip imported
+		if acct == "imported" {
+			continue
+		}
+		// There is no construct for getmasterpubkey in dcrd yet so use raw json
+		rawreq := []json.RawMessage{json.RawMessage(`"` + acct + `"`)}
```

Comment:
I'm not totally comfortable with using raw requests here. A better solution would be to get the RPC added to dcrwallet properly.

Reply:
oh man, do you think I could get a commit in dcrd? hell yeah!

Reply:
awaiting merge in dcrd